src/store.ts

```diff
@@ -17,6 +18,11 @@ export const isShowingPoints = writable(defaults.showPoints);
 export const initialViewport = writable(defaults.viewport);
 export const navMode = writable(NavMode.autofit);
 
+export const formSelections = writable(getFormSelections());
+formSelections.subscribe((val) => {
```

Comment:
Should this register a handler to unsubscribe when it goes out of scope or gets collected or whatever?

Reply:
That should be handled automatically.